x/pool/keeper/keeper.go

```diff
@@ -0,0 +1,71 @@
+package keeper
+
+import (
+	"context"
+	"fmt"
+
+	storetypes "cosmossdk.io/core/store"
+	"cosmossdk.io/log"
+	"cosmossdk.io/x/pool/types"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type Keeper struct {
+	// Msg server router
+	router baseapp.MessageRouter
```

Comment:
Where is this used?

Reply:
it should be removed, added it during initial trials